.eslintrc.json

```diff
@@ -45,6 +45,25 @@
         "ignoreRestSiblings": true
       }
     ],
+    "@typescript-eslint/member-ordering": [
```

Comment:
I think this should go before `@typescript-eslint/no-use-before-define` - alphabetical rules